content/docs/faq-structure.md

```diff
@@ -59,16 +59,18 @@ components/
   ProfileHeader.css
 ```
 
-Some people also prefer to go further, and separate components into different folders depending on their role in the application. For example, [Atomic Design](http://bradfrost.com/blog/post/atomic-web-design/) is a design methodology built on this principle. Remember that it's often more productive to treat such methodologies as helpful examples rather than strict rules to follow.
+어떤 사람들은 더 나아가서 해당 애플리케이션 내에서의 역할에 따라 컴포넌트를 다른 폴더로 분리하는 것을 선호합니다. 예를 들어 [아토믹 디자인](http://bradfrost.com/blog/post/atomic-web-design/)은 이와 같은 원칙에 기반한 디자인 방법론입니다. 지켜야 할 엄격한 규칙이라기보다 도움이 되는 예시로 이러한 방법론을 활용하는 것은 종종 더욱 생산적이라는 사실을 잊지 마세요.
 
-#### Avoid too much nesting {#avoid-too-much-nesting}
+#### 너무 많은 중첩을 피하세요 {#avoid-too-much-nesting}
 
-There are many pain points associated with deep directory nesting in JavaScript projects. It becomes harder to write relative imports between them, or to update those imports when the files are moved. Unless you have a very compelling reason to use a deep folder structure, consider limiting yourself to a maximum of three or four nested folders within a single project. Of course, this is only a recommendation, and it may not be relevant to your project.
+자바스크립트 프로젝트에서 깊은 디렉토리 중첩과 관련된 문제점들이 많이 있습니다. 프로젝트 간에 상대 경로를 통한 임포트를 작성하거나 또는 파일이 옮겨졌을 때 그러한 임포트들을 업데이트하는 것이 더 어려워집니다. 깊은 폴더 구조를 사용해야 하는 매우 설득력 있는 이유가 없다면, 단일 프로젝트 내에서는 3번 혹은 4번을 최대한으로 폴더를 중첩하도록 제한하는 것을 고려해보세요.
```

Comment:
- `자바스크립트` -> `JavaScript`
- `디렉토리`에 대해서는 [엄밀히 말한다면 `폴더`와는 차이점이 있다고는 하지만](https://en.wikipedia.org/wiki/Directory_%28computing%29#Folder_metaphor) 해당 문서의 맥락에서는 문서 전반에서 사용하는 `폴더`로 수정해도 무리가 없어 보이는데 어떻게 생각하실까요?
- 문장 끝에 `Of course, this is only a recommendation, and it may not be relevant to your project.`에 대한 번역이 누락되었어요. 추가 부탁드려요.

Reply:
- 고유명사인 JavaScript는 그대로 남겨두겠습니다.

- 이미 `directory`라는 용어가 쓰이기 이전에 ` folder`가 4번이나 쓰이긴 했습니다. 그리고 첨부해주신 링크 통해서 디렉토리와 폴더의 차이도 약간 살펴보았습니다. 그래도 굳이 저자가 `directory`를 사용한 이유가 있을 것 같아서, 디렉토리를 사용하는게 개인적으로 더 낫다고 생각합니다. 

- 죄송합니다. 마지막 문장에 대한 번역 추가했습니다.

Reply:
> 이미 directory라는 용어가 쓰이기 이전에 folder가 4번이나 쓰이긴 했습니다. 그리고 첨부해주신 링크 통해서 디렉토리와 폴더의 차이도 약간 살펴보았습니다. 그래도 굳이 저자가 directory를 사용한 이유가 있을 것 같아서, 디렉토리를 사용하는게 개인적으로 더 낫다고 생각합니다.

음 여기서는 어떤 선택을 해도 무리가 없어보여요. 제안해주신 `디렉토리`로 갑시다 :)